src/lib/Components/Inbox/Conversations/Messages.tsx

```diff
@@ -203,7 +202,7 @@ export default createPaginationContainer(
       }
     },
     query: graphql.experimental`
-      query MessagesPaginationQuery($conversationID: String!, $count: Int!, $after: String) {
+      query MessagesQuery($conversationID: String!, $count: Int!, $after: String) {
```

Comment:
For my information, was this needed or just a cleanup?

Reply:
Nope, just cleanup.